key/src/main/java/software/amazon/kms/key/BaseHandlerStd.java

```diff
@@ -127,7 +127,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> softFailAccessDenied(
         final CallbackContext callbackContext) {
         try {
             return eventSupplier.get();
-        } catch (final CfnAccessDeniedException e) {
+        } catch (final CfnAccessDeniedException | CfnUnauthorizedTaggingOperationException e) {
             return ProgressEvent.progress(model, callbackContext);
         }
```

Comment:
I believe we need to returned `failed` progress event and not `progress`

Reply:
This is only used in the read handler to make sure we READ other attributes if one of the Get calls throws access denied (For example due to the key policy or something).